### PR TITLE
[DUOS-1000][risk=no] Standardize DatasetDAO

### PR DESCRIPTION
**Pull request by connorlbark**
## Addresses

https://broadworkbench.atlassian.net/browse/DUOS-1000

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java [hunk 1]
```diff
@@ -108,102 +118,118 @@ Integer insertDataset(
...
     @UseRowMapper(DatasetDTOWithPropertiesMapper.class)
-    @SqlQuery(" SELECT d.*, k.key, dp.propertyvalue, ca.consentid, c.dac_id, c.translateduserestriction " +
-            " FROM dataset d INNER JOIN datasetproperty dp ON dp.datasetid = d.datasetid INNER JOIN dictionary k ON k.keyid = dp.propertykey " +
-            " INNER JOIN consentassociations ca ON ca.datasetid = d.datasetid INNER JOIN consents c ON c.consentid = ca.consentid " +
-            " WHERE d.datasetid IN (<dataSetIdList>) ORDER BY d.datasetid, k.receiveorder ")
+    @SqlQuery(
+        "SELECT d.*, k.key, dp.property_value, ca.consentid, c.dac_id, c.translateduserestriction " +
+            " FROM dataset d INNER JOIN dataset_property dp ON dp.dataset_id = d.dataset_id INNER JOIN dictionary k ON k.key_id = dp.property_key " +
+            " INNER JOIN consentassociations ca ON ca.datasetid = d.dataset_id INNER JOIN consents c ON c.consentid = ca.consentid " +
+            " WHERE d.dataset_id IN (<dataSetIdList>) ORDER BY d.dataset_id, k.receive_order ")
```
Comment (shaemarks) [inline src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java:238]: Formatting could be improved here by adding newlines

**Review by shaemarks — APPROVED** [review]
Tested locally and name changes look good 👍 

minor optional nit: some of the queries in DatasetDAO still have a space before SELECT

File: src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java (hunk 1 above)
Comment (connorlbark) [reply src/main/java/org/broadinstitute/consent/http/db/DatasetDAO.java:238]: oops that happened with the merge, good catch!

**Review by JVThomas — APPROVED** [review]
Looks good to me. Thanks for the column name updates as well!

**Pull request merged**